libs/pages/applications/feature/src/lib/applications-page.tsx

```diff
@@ -1,12 +1,14 @@
-import { useApplications } from '@console/domains/environment'
 import { useDocumentTitle } from '@console/shared/utils'
 import { Container } from '@console/pages/applications/ui'
+import { useParams } from 'react-router'
+import { useApplications } from '@console/domains/application'
 
 export function ApplicationsPage() {
   useDocumentTitle('Applications - Qovery')
-  const { applications } = useApplications()
+  const { applications, applicationsByEnv } = useApplications()
+  const { environmentId } = useParams()
 
-  return <Container applications={applications} />
+  return <Container applications={applicationsByEnv(environmentId || '')} />
```

Comment:
same here, we can't render the `<Container />` component if we don't have environmentId